tools/profiling/memory/memory_diff.py

```diff
@@ -175,20 +179,37 @@ def _run():
                     text += "{}: {}\n".format(key, cur[key])
                 else:
                     text += "{}: {} -> {}\n".format(key, old[key], cur[key])
+                    diff = cur[key] - old[key]
                     if "call" in key:
-                        call_diff_size += cur[key] - old[key]
+                        if diff < -call_significant:
+                            call_decrease += 1
+                        elif diff > call_significant:
+                            call_increase += 1
                     else:
-                        channel_diff_size += cur[key] - old[key]
-
-    print("CALL_DIFF_SIZE: %f" % call_diff_size)
-    print("CHANNEL_DIFF_SIZE: %f" % channel_diff_size)
-    check_on_pr.label_increase_decrease_on_pr(
-        "per-call-memory", call_diff_size, 64
-    )
-    check_on_pr.label_increase_decrease_on_pr(
-        "per-channel-memory", channel_diff_size, 1000
-    )
-    # TODO(chennancy)Change significant value when minstack also runs for channel
+                        if diff < -channel_significant:
+                            channel_decrease += 1
+                        elif diff > channel_significant:
+                            channel_increase += 1
+
+    print("CALL: %d increases, %d decreases" % (call_increase, call_decrease))
+    print("CHANNEL: %d increases, %d decreases" % (channel_increase, channel_decrease))
+    # if anything increased ==> label it an increase
+    # otherwise if anything decreased (and nothing increased) ==> label it a decrease
+    # otherwise label it neutral
+    # -- this biases reporting towards increases, which is what we want to act on
+    #    most regularly
+    if call_increase > 0:
+        check_on_pr.label_significance_on_pr("per-call-memory", 1)
```

Comment:
This will always show a label with low significance. Did you consider mapping the significant to the number of call increases? `check_on_pr.label_significance_on_pr("per-call-memory", min(call_increase, 3))`.

Reply:
Updated to use the increase/decrease thing (I had done that before I sent the review, but github stopped accepting pushes for a bit)